Composer/packages/adaptive-form/src/components/ExpressionSwitchWindow.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { FontIcon } from 'office-ui-fabric-react/lib/Icon';
+import React from 'react';
+import formatMessage from 'format-message';
+
+const styles = {
+  fieldTypeText: css`
+    height: 32px;
+    font-style: italic;
+    color: #8a8886;
+    padding: 0 4px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    font-size: 15px;
+  `,
+  switchToExpressionText: css`
+    height: 32px;
+    cursor: pointer;
+    padding: 0 4px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    font-size: 15px;
+  `,
+  icon: css`
+    margin-right: 5px;
+  `,
+};
+
+export const ExpressionSwitchWindow = (props: { type: string; onSwitchToExpression: () => void }) => {
```

Comment:
Nit: It is much cleaner to define a type Props rather than have it inline, even when only 2 properties.

Reply:
ok